packages/server/test/support/helpers/e2e.js

```diff
@@ -423,6 +422,11 @@ const e2e = {
 
       Fixtures.scaffold()
 
+      if (process.env.EXIT === 'false') {
```

Comment:
This doesn't work if `exit: false` is passed in as an option and not via the command line. For example:

```javascript
e2e.exec(this, {
  spec: 'app_spec.js',
  exit: false,
})
// or
e2e.it('some test', {
  spec: 'app_spec.js',
  exit: false,
})
```

I see that this is because this code is in the setup, which is before `e2e.exec()` or `e2e.it()` is called, but I wonder if we can support it via those methods too.

Reply:
since `exit` is an option that's only useful for debugging, it should really only be passed via the cli

Reply:
and e2e.exec will only run in electron and ignores `--browser chrome`, so that's why the browser never launched

Reply:
> since exit is an option that's only useful for debugging, it should really only be passed via the cli

I frequently used `exit: false` via `e2e.exec`. I often prefer it to fiddling with command line options. 

> and e2e.exec will only run in electron and ignores --browser chrome, so that's why the browser never launched

I've been using `browser: 'chrome'` a lot in the error improvements branch with an `e2e.exec` test. The behavior seems to have changed with this PR.

Reply:
I see your point about only using `exit` for debugging via the cli, but I think we should either fully support `exit: false` as an option or restrict it to only cli usage. It's not good to have it work in some cases (`e2e.it`) and not others (`e2e.exec`). If we only accept it via cli, we should error or at least warn if it's used via the programmatic API.